samples/excel/85-preview-apis/pivottable-pivotfilters.yaml

```diff
@@ -55,10 +55,10 @@ script:
               dateHierarchy = pivotTable.rowHierarchies.add(pivotTable.hierarchies.getItem("Date Updated"));
             }
 
-            // Apply a date filter for anything logged before August.
+            // Apply a date filter to fliter out anything logged before August.
             const filterField = dateHierarchy.fields.getItem("Date Updated");
             const dateFilter = {
-              condition: Excel.DateFilterCondition.before,
+              condition: Excel.DateFilterCondition.afterOrEqualTo,
```

Comment:
I think something is off here, clicking the August 2020 button is also displaying a bunch of July dates.